tests/integ/scala/test_snowflake_plan_suite.py

```diff
@@ -124,7 +124,10 @@ def test_multiple_queries(session):
 )
 def test_plan_height(session, temp_table, sql_simplifier_enabled):
     df1 = session.table(temp_table)
-    assert df1._plan.plan_height == 1
+    if sql_simplifier_enabled:
```

Comment:
This is from the fix on Table constructor. When sql simplifier is enabled df1._plan is `SelectStatement -> SelectEntity`